core/models/asset.py

```diff
@@ -247,11 +255,32 @@ def save(self, *args, **kwargs):
         super(AssetCondition, self).save(*args, **kwargs)
 
 
+class AssetIncidentReport(models.Model):
```

Comment:
i think we might need to use `TextField ` for some of these fields.. attaching the form used on the app.

Reply:
working on it